server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultClusterContext.java

```diff
@@ -125,6 +125,12 @@ public DefaultClusterContext(String name,
         this.clusterMembers.addAll(members);
         this.schedulerProvider = schedulerProvider;
         this.connectOptions = connectOptions;
+
+        if (!GL.isTraceEnabled(GL.CLUSTER_LOGGER_NAME)) {
+            // Note: must use Logger.getLogger, not LogManager.getLogger
```

Comment:
Will this affect the logging that users of our cluster feature expect.  Perhaps we can reflect the same log level instead of turn on vs off...?